adding onto our feature request template (#5800)
DataDog/documentation: .github/ISSUE_TEMPLATE/feature_request.md

---
name: Feature request

> about: Suggest an idea for the Datadog documentation. To request a product feature, please [file a ticket with our support team](https://docs.datadoghq.com/help/)

Reviewer: ```suggestion
about: Suggest an idea for the Datadog documentation. To request a product feature, please [file a ticket to Datadog support team directly](https://docs.datadoghq.com/help/).
```
